doc/en/user/source/data/database/postgis.rst

```diff
@@ -59,10 +86,63 @@ To begin, navigate to :menuselection:`Stores --> Add a new store --> PostGIS NG`
      - Time (in seconds) the connection pool will wait before timing out.
    * - :guilabel:`validate connections`
      - Checks the connection is alive before using it.
+   * - :guilabel:`Evictor run periodicity`
+     - Number of seconds between idle object evictor runs.
+   * - :guilabel:`Max connection idle time`
+     - Number of seconds a connection needs to stay idle before the evictor starts to consider closing it.
+   * - :guilabel:`Evictor tests per run`
+     - Number of connections checked by the idle connection evictor for each of its runs.
+
+Connection parameters managing SQL generation:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Connection Parameter
+     - Description
+   * - :guilabel:`Expose primary keys`
+     - Expose primary key columns as values suitable for filtering.
+   * - :guilabel:`Primary key metadata table`
+     - Provide table defining how primary keys values are generated (see :doc:`primarykey`)     
+   * - :guilabel:`Session startup SQL`
+     - SQL applied to connection before use (see :doc:`sqlsession`)
+   * - :guilabel:`Session close-up SQL`
+     - SQL applied to connection after use (see :doc:`sqlsession`)
+   * - :guilabel:`preparedStatements`
+     - Enables prepared statements for SQL generation, rather than text substitution.
+   * - :guilabel:`Max open prepared statements`
+     - Number of prepared statements available.
+
+Connection parameters managing database interaction:
+
+.. list-table::
+   :widths: 20 80
+   :header-rows: 1
+
+   * - Connection Parameter
+     - Description
    * - :guilabel:`Loose bbox`
      - Performs only the primary filter on the bounding box.  See the section on :ref:`postgis_loose_bbox` for details.
-   * - :guilabel:`preparedStatements`
-     - Enables prepared statements.
+   * - :guilabel:`Estimated extends`
```

Comment:
It matches the screenshot, but I think this should be `Estimated extents`. Another ticket?